internal/cfgplugins/policyforwarding.go

```diff
@@ -494,6 +498,10 @@ func DecapGroupConfigGre(t *testing.T, dut *ondatra.DUTDevice, pf *oc.NetworkIns
 		switch dut.Vendor() {
 		case ondatra.ARISTA:
 			helpers.GnmiCLIConfig(t, dut, decapGroupGREArista)
+			if ocPFParams.Dynamic {
+				t.Logf("Going into decap")
+				GreDecapConfig(t, dut, ocPFParams)
+			}
```

Comment:
aren't these mutually exclusive?

```suggestion
		case ondatra.ARISTA:
			if ocPFParams.Dynamic {
				t.Logf("Going into decap")
				GreDecapConfig(t, dut, ocPFParams)
			}
			else {
				helpers.GnmiCLIConfig(t, dut, decapGroupGREArista)
			}
```